### ISSUE TITLE
[feature request] More awesome fonts

### ISSUE THREAD

**Issue by shibumi** [gabrielelana/awesome-terminal-fonts#26]
Hello,
I've just found out that awesome-terminal-fonts only support characters until \uf196.
Could you please add the other characters?

This is a list of other characters:

![other characters](https://paste.archlinux.de/ruv1Q/)

You can find more from it here:
https://github.com/ryanoasis/nerd-fonts

**Comment by gabrielelana** [comment]
I looked into `ryanoasis/nerd-fonts` they include almost the same icon fonts

You don't have to look at the code point of the glyph (aka `\uf196`) to search for it because code points for icon glyphs are not fixed since they are not part of the UNICODE standard.

I relocate the glyphs to avoid clashing between icon fonts, see `https://github.com/gabrielelana/awesome-terminal-fonts/blob/master/scripts/fu-relocate`

**Comment by shibumi** [comment]
@gabrielelana Ok, So how I got for example one of this fancy firefox icons? I can't find it in your icons.
You said that you just relocated it. So it has some other unicode? how can I print all your symbols with the unicode for it? do you have some script for it?

**Comment by gabrielelana** [comment]
@shibumi Thanks to you I found a problem, the `font-awesome` doesn't correctly define the glyphs inside the font. In the latest commits I've updated `font-awesome` to the latest release and used the CSS to create the font maps, so if you install the current master `echo "\u$CODEPOINT_OF_AWESOME_FIREFOX"` should work

**Comment by shibumi** [comment]
@gabrielelana Awesome! Do you have an overview over all icons, like a file there you map the unicodes to names for the icons or something?

**Comment by gabrielelana** [comment]
@shibumi sure, for each font file there's a font map file like [this](https://github.com/gabrielelana/awesome-terminal-fonts/blob/master/build/fontawesome-regular.sh) all you have to do is to source those files and then you can refer to the glyphs by name

**Comment by shibumi** [comment]
Thanks!

**Comment by shibumi** [comment]
@gabrielelana I have tried it now with the newest master and font-awesome.
The mapping seems wrong:

In the file is firefox `\uf269` but if I try this, I get the beer icon..

**Comment by gabrielelana** [comment]
@shibumi I've checked, for me it works and looking at the font with `fontforge` the firefox glyph is at `\uf269`. Can you try `fc-cache -fv ~/.fonts`? If it still doesn't work, try to restart X

**Comment by shibumi** [comment]
@gabrielelana 
Hello, Ok I have restarted X now. It hasn't changed something. And I've recognized another problem.
All icons are now wrong in my powerline.

Do you see the green icon there? normally there should be an 'ok'-sign.
![bad icon](https://paste.archlinux.de/Bs6/) 

And my git commit icon is now a 'female icon'.

Seems like your last commits have missed up the usual encodings and all unicodes are now different.

**Comment by gabrielelana** [comment]
@shibumi You should always refer to glyphs by name on not by codepoint because when I update the fonts I may have to shuffle things around to avoid conflicts between all the fonts. You can find all the font names in the font maps. In your case try `echo -e "\u$CODEPOINT_OF_AWESOME_CHECK"`

**Comment by shibumi** [comment]
Hello,
am talking about for example this icon here:

`CODEPOINT_OF_AWESOME_MEDIUM='f23a'`

In your font it's now a green M in my airline bar it should be an OK icon. It was an OK icon until commit 73f8840

**Comment by davidmpaz** [comment]
Hi there, 

is there any results here? 

I have the same problem. I have check the `powerlevel9k` theme also. But for example when I try on my terminal: `echo '\f00c'` which is the unicode for the check mark I get a weird character I can not recognize, or maybe the number 8, here is it:
![selection_075](https://cloud.githubusercontent.com/assets/210742/21176462/1cc7e7ac-c1e7-11e6-9bd0-f8e95efd8022.png)

I have installed with the fallback strategy, by executing the shell scripts in the repo.

Thanks in advance for such a good project. My terminal looks really niceer now ;)

**Comment by shibumi** [comment]
@davidmpaz 
Which version of awesome-terminal-fonts do you use? I have no issues with git commit: fcfef52...

**Comment by davidmpaz** [comment]
Hi @shibumi , sorry for coming back late I have tried to use that commit. But still the same result. 

I'm thinking now whether I messed with my system in a way I can not get it stable now with the fonts. Here is my output when installing:
![selection_078](https://cloud.githubusercontent.com/assets/210742/21607260/4aeea44e-d1b5-11e6-8225-0c1e9b182a11.png)

**Comment by shibumi** [comment]
Hey @davidmpaz 
The powerline looks ok on your screenshot. So you have only trouble with the 'OK'-Mark?
I am installing the fonts via Arch Linux package.  I don't call install.sh or build.sh
I just move the fonts in the right directory. Have a look at:
```
 install -d "${pkgdir}/usr/share/fonts/${_pkgname}"
  install -d "${pkgdir}/etc/fonts/conf.avail"
  cd "${srcdir}/${_pkgname}/build"
  install -p -m 0644 * "${pkgdir}/usr/share/fonts/${_pkgname}/"
  install -p -m 0644 "${srcdir}/${_pkgname}/config/10-symbols.conf" "${pkgdir}/etc/fonts/conf.avail"
```

You can ignore the ${pkgdir} and ${_pkgname} stuff thats arch specific. 
I just go to the build dir and move everything in it to /usr/share/fonts/awesome-terminal-fonts/
and then I move the 10-symbols.conf to /etc/fonts/conf.avail

and after all of that I update the fontfache via fc-cache

**Comment by davidmpaz** [comment]
Hi @shibumi 
thanks for the hints, I did all what you said with respective recognized correct commit, unfortunately the result is the same. But I have some other news, by coincidence I found the OK icon!!

So, in my case and in terms of icons, what should be: `CODEPOINT_OF_AWESOME_OK='f00c'` is actually: `CODEPOINT_OF_POMICONS_PAIR_PROGRAMMING='e007'` I think this confirm #31 issue, the font mappings are somehow broken.
![selection_079](https://cloud.githubusercontent.com/assets/210742/21677043/a57eb848-d338-11e6-82da-237add94c24e.png)
From my side this issue is not relevant anymore, I think I have the fonts properly installed, I just can not find the correct unicode value for the icon.

Thanks again!
Regards
